src/test/java/lotto/model/LottoTest.java

```diff
@@ -0,0 +1,28 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class LottoTest {
+
+    @Test
+    @DisplayName("로또는 6개의 숫자를 가진다.")
+    void lottoContains6NumsersTest() {
+        Lotto lotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
        Lotto lotto = new Lotto(1, 2, 3, 4, 5, 6);
```

자바의 가변 인자를 통해 위와 같이 구현할 수 있다면 테스트 코드가 더 깔끔해지지 않을까?